modules/layers/src/text-layer/text-layer.js

```diff
@@ -201,14 +203,22 @@ export default class TextLayer extends CompositeLayer {
 
       for (const object of iterable) {
         objectInfo.index++;
-        const text = getText(object, objectInfo) || '';
+        const text = Array.from(getText(object, objectInfo) || '');
+        if (autoCharacterSet) {
+          text.forEach(autoCharacterSet.add, autoCharacterSet);
+        }
         // When dealing with double-length unicode characters, `str.length` is incorrect
```

Comment:
Is this comment now obsolete?

Reply:
No, we still are not using string.length